src/all/batoto/src/eu/kanade/tachiyomi/extension/all/batoto/BatoTo.kt

```diff
@@ -392,7 +393,11 @@ open class BatoTo(
     }
 
     private fun checkChapterLists(document: Document): Boolean {
-        return document.select(".episode-list > .alert-warning").text().contains("This comic has been marked as deleted and the chapter list is not available.")
+        val alertWarningElement = document.select(".episode-list > .alert-warning").firstOrNull()
+        if (alertWarningElement != null && alertWarningElement.text().contains("This comic has been marked as deleted and the chapter list is not available.")) {
+            throw IOException("This comic was deleted.")
+        }
+        return false
```

Comment:
check if chapter list is empty and then check for this and throw

Reply:
Done